demisto_client/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 
 def configure(base_url=None, api_key=None, verify_ssl=None, proxy=None, username=None, password=None,
-              ssl_ca_cert=None, debug=False):
+              ssl_ca_cert=None, connection_pool_maxsize=None, debug=False):
```

Comment:
add after debug so we don't break the API
```suggestion
              ssl_ca_cert=None, debug=False, connection_pool_maxsize=None):
```